codecov.yml

```diff
@@ -0,0 +1,17 @@
+codecov:
```

Comment:
wouldn't `webpack-defaults` just create this file here anyway? Or, do u mean it also needs to be added there?